Configuration/Applications/scripts/cmsDriver.py

```diff
@@ -14,17 +13,6 @@ def run():
         # after cleanup of all config parameters pass it to the ConfigBuilder
         configBuilder = ConfigBuilder(options, with_output = True, with_input = True)
 
-        # Switch on any eras that have been specified. This is not required to create
```

Comment:
Where did the era handling go? Or was this not actually needed?

Reply:
It was obsoleted by #39127.